files/build_templates/init_cfg.json.j2

```diff
@@ -17,33 +17,34 @@
 {% endfor %}
         }
     },
-{%- set features = [("bgp", "enabled", false, "enabled"),
-                   ("database", "enabled", false, "disabled"),
-                   ("dhcp_relay", "enabled", false, "enabled"),
-                   ("lldp", "enabled", false, "enabled"),
-                   ("pmon", "enabled", false, "enabled"),
-                   ("radv", "enabled", false, "enabled"),
-                   ("snmp", "enabled", true, "enabled"),
-                   ("swss", "enabled", false, "enabled"),
-                   ("syncd", "enabled", false, "enabled"),
-                   ("gbsyncd", "enabled", false, "enabled"),
-                   ("teamd", "enabled", false, "enabled")] %}
-{%- if include_iccpd == "y" %}{% do features.append(("iccpd", "disabled", false, "enabled")) %}{% endif %}
-{%- if include_mgmt_framework == "y" %}{% do features.append(("mgmt-framework", "enabled", true, "enabled")) %}{% endif %}
-{%- if include_nat == "y" %}{% do features.append(("nat", "disabled", false, "enabled")) %}{% endif %}
-{%- if include_restapi == "y" %}{% do features.append(("restapi", "enabled", false, "enabled")) %}{% endif %}
-{%- if include_sflow == "y" %}{% do features.append(("sflow", "disabled", false, "enabled")) %}{% endif %}
-{%- if include_system_telemetry == "y" %}{% do features.append(("telemetry", "enabled", true, "enabled")) %}{% endif %}
+{%- set features = [("bgp", "enabled", false, "enabled", "local"),
+                   ("database", "enabled", false, "disabled", "local"),
+                   ("dhcp_relay", "enabled", false, "enabled", "kube"),
+                   ("lldp", "enabled", false, "enabled", "kube"),
+                   ("pmon", "enabled", false, "enabled", "kube"),
+                   ("radv", "enabled", false, "enabled", "kube"),
+                   ("snmp", "enabled", true, "enabled", "kube"),
```

Comment:
yes, my thinking is that only when include_kuber is define, if not, we should still say local.

Reply:
Yep. No change, unless switch connects to kube master and there is a manifest deployed.